src/app/helpers/tests/jest-setup.js

```diff
@@ -1,3 +1,5 @@
 import fetch from 'jest-fetch-mock'; // eslint-disable-line import/no-extraneous-dependencies
 
+// ignore console.error() in testing suite to silence node_dependency console statements
+global.console.error = jest.fn();
```

Comment:
This line also silences PropType errors. Did you intend this to happen?

Reply:
e.g. before:
```
PASS  src/app/components/Headline/index.test.jsx
  ● Console

    console.error node_modules/prop-types/checkPropTypes.js:19
      Warning: Failed prop type: Invalid prop `blocks[0]` supplied to `Headline`.
          in Headline (at index.test.jsx:8)
```
After: no error.

Reply:
Yep that was intended as none of our node dependencies should be logging during our test suite unless that have valid errors

Reply:
I'd suggest using this instead:
```
console.error = jest.fn(error => {
  throw new Error(error);
});
```

Reply:
As we spoke F2F adding here for historical reasons, there is actually an underlying issue here where the dependency prop-types only logs a console.error instead of throwing a real error. Therefore I've opened the issue https://github.com/bbc/simorgh/issues/289 to handle this.

Reply:
Considering the issue above I'm going to remove this line as it can be handled in #289